### ISSUE TITLE
Accuracy without finetune on CIFAR10

### ISSUE THREAD

**Issue by Adamdad** [google-research/google-research#722]
I am implementing this paper myself. Without fine-tuning, the student trained on pseudo-label (CIFAR-10-4000) only achieves an accuracy of about 50%. Is this the true status or this is my implementation issue? What is the official accuracy without fine-tuning?
Thank you so much!
